api/pom.xml

```diff
@@ -209,6 +209,11 @@
 			<groupId>org.hibernate</groupId>
 			<artifactId>hibernate-validator</artifactId>
 	   </dependency>
+	   <dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
```

Comment:
@reagan-meant   dont we need to define a version for this?

Reply:
did you take a look at this. https://issues.openmrs.org/browse/TRUNK-5558

Reply:
it is already defined in openmrs pom otherwise it gives an error